user/deployment/testfairy.md

```diff
@@ -0,0 +1,99 @@
+---
+title: TestFairy deployment
+layout: en
+permalink: /user/deployment/testfairy/
+---
+
+Travis CI can automatically deploy your Android and iOS Apps to [TestFairy](https://www.testfairy.com/).
+
+For a minimal configuration, add the following `deploy` key to your `.travis.yml`:
+
+``` yaml
+deploy:
+  provider: testfairy
+  api-key: "TESTFAIRY API KEY"
+  app-file: Path to the app file (APK/IPA)
+```
+
+You can find your API key on [TestFairy settings page](https://app.testfairy.com/settings/).
+
+It is recommended that you encrypt your api-key. If you have the Travis CI command line client installed, you can run the following command in your repository directory:
+
+``` console
+$ travis encrypt "YOUR API KEY" --add deploy.api-key
+```
+
+## Android
+
+To deploy your Android application you need to specify your keystore certificate:
+
+``` yaml
+deploy:
+  provider: testfairy
+  api-key: "TESTFAIRY API KEY"
+  app-file: Path to the app file (APK/IPA)
+  keystore-file: Path to your keystore-file
+  storepass: storepass
+  alias: alias
+```
```

Comment:
we should call the file the certificate if that is what it is. What is the alias? is it required? The pass is the pass to the ecrypted file?